### PR TITLE
quick fix for CRAN

### PR DESCRIPTION
**Pull request by BenGraeler**

### CODE REVIEW

File: tests/testthat/test_getobservation.R [hunk 1]
```diff
@@ -624,7 +624,7 @@ test_that("POX with offering object in list and not in list (2.0.0)", {
 # FOI retrieval can be disabled (tested with POX, 2.0.0) ----
 #
 test_that("FOI retrieval can be disabled (tested with POX, 2.0.0)", {
-  #skip_on_cran()
+  skip_on_cran()
```
Comment (nuest) [inline tests/testthat/test_getobservation.R:627]: I don't think that unskipping this test solves a problem on CRAN. I suggest to skip more, not less tests :-)

File: vignettes/sos4R.Rnw.old [hunk 2]
```diff
@@ -1,1875 +0,0 @@
-\documentclass[nogin,a4paper]{article}
```
Comment (nuest) [inline vignettes/sos4R.Rnw.old:1]: What's the reason to remove this file? I agree it's not really helpful anymore, but wondering why as part of this PR.

File: tests/testthat/test_getobservation.R (hunk 1 above)
Comment (BenGraeler) [reply tests/testthat/test_getobservation.R:627]: that's what we do

File: vignettes/sos4R.Rnw.old (hunk 2 above)
Comment (BenGraeler) [reply vignettes/sos4R.Rnw.old:1]: it is pure coincidence, no particular relation to this PR

File: tests/testthat/test_getobservation.R (hunk 1 above)
Comment (nuest) [reply tests/testthat/test_getobservation.R:627]: Ahh, sorry. I should practice reading diffs..

**Pull request merged**